src/main/webapp/i18n/de/programmingExercise.json

```diff
@@ -189,6 +189,7 @@
             "createBehavioralSolutionEntriesTooltip": "Erstellt Lösungs Codeschnipsel für alle Verhaltenstests dieser Aufgabe.",
             "createBehavioralSolutionEntriesTitle": "Erstelle verhaltensweise Lösungs Codeschnipsel",
             "createBehavioralSolutionEntriesSuccess": "Verhaltensweise Lösungs Codeschnipsel wurden erfolgreich erstellt",
+            "studentView": "Studentenansicht",
```

Comment:
The translation was also added for the button in the course overview (#7597 ). Since we use this translation multiple times in different views, I would remove both and add a single entry in global.json --> entity --> action